### PR TITLE
Create python-app.yml

### PR DESCRIPTION
**Pull request by Erez-Shlingbaum**

### CODE REVIEW

**Comment by Erez-Shlingbaum** [comment]
We should merge tests branch before this

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
```
Comment (segalmatan) [inline .github/workflows/python-app.yml]: Are we using this version?

**Review by segalmatan — CHANGES_REQUESTED** [review]

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    branches: [ dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.10
```
Comment (segalmatan) [inline .github/workflows/python-app.yml]: fix version

Comment (segalmatan) [reply .github/workflows/python-app.yml]: why have the version in the name?

**Review by segalmatan — APPROVED** [review]

**Pull request merged**